### PR TITLE
fix(LH-70705): Fix FTD onboarding fails when ftd is already register

### PR DESCRIPTION
**Pull request by weilueluo**
https://jira-eng-rtp3.cisco.com/jira/browse/LH-70705

### CODE REVIEW

File: client/device/cloudftd/cloudftdonboarding/create.go [hunk 1]
```diff
@@ -69,10 +103,6 @@ func Create(ctx context.Context, client http.Client, createInp CreateInput) (*Cr
 	client.Logger.Println("creating FTD device record in FMC")
 
 	// 3.1 read ftd metadata
-	readFtdOutp, err := cloudftd.ReadByUid(ctx, client, cloudftd.NewReadByUidInput(createInp.FtdUid))
-	if err != nil {
-		return nil, err
-	}
```
Comment (weilueluo) [inline client/device/cloudftd/cloudftdonboarding/create.go:75]: just moving this to above

File: client/model/cloudfmc/accesspolicies/accesspolicies.go
Comment (weilueluo) [inline client/model/cloudfmc/accesspolicies/accesspolicies.go:1]: changes in this file are just refactors

File: provider/examples/resources/ftd/cdo_ftd.tf [hunk 2]
```diff
@@ -5,7 +5,7 @@ terraform {
     }
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.27.0"
+      version = "~> 3.75.0"
```
Comment (weilueluo) [inline provider/examples/resources/ftd/cdo_ftd.tf:8]: pump version because 3.27 does not have support for m1

**Review by siddhuwarrier — APPROVED** [review]
Nice work!

**Pull request merged**